lib/platform/gitlab/index.ts

```diff
@@ -99,7 +99,17 @@ export async function initPlatform({
 export async function getRepos(): Promise<string[]> {
   logger.debug('Autodiscovering GitLab repositories');
   try {
-    const url = `projects?membership=true&per_page=100&with_merge_requests_enabled=true&min_access_level=30`;
+    const query = new URLSearchParams({
+      membership: 'true',
+      per_page: '100',
+      with_merge_requests_enabled: 'true',
+      min_access_level: '30',
+    });
+    const lastActivityAfter = process.env.GITLAB_LAST_ACTIVITY_AFTER;
+    if (lastActivityAfter != null) {
```

Comment:
Can we do some more verification on this? The GitLab API says it should be in the format `ISO 8601 YYYY-MM-DDTHH:MM:SSZ`

Reply:
Thanks! Would a little helper for this make sense to keep it clean? Any other place to put this or keep it in this file if it's only used for gitlab? something like
```typescript
function isValidIso8601(date: string): boolean {
  return moment(date, moment.defaultFormat, true).isValid();
}
```
I guess I'll also log a warning that the option is ignored if it exists but isn't valid.